src/views/TopologyDisplayView.vue

```diff
@@ -7,7 +7,7 @@
       :items="items"
     />
   </Teleport>
-  <Teleport to="#web-oc-toolbar-target">
+  <Teleport to="#web-oc-toolbar-target" v-if="showWorkflowsControl">
     <WorkflowsControl
```

Comment:
Maybe the check should go on the control itself, in case we get more controls here